### PR TITLE
Improve web forms

### LINKED ISSUE #456: Make web forms usable and presentable
- [x] Group: https://skohub.io/editor/?schema=https://hbz.github.io/lobid/schemas/group.json
- [x] Person: https://skohub.io/editor/?schema=https://hbz.github.io/lobid/schemas/person.json
- [x] Product: https://skohub.io/editor/?schema=https://hbz.github.io/lobid/schemas/product.json
- [x] Project: https://skohub.io/editor/?schema=https://hbz.github.io/lobid/schemas/project.json
- [x] Publication: https://skohub.io/editor/?schema=https://hbz.github.io/lobid/schemas/publication.json

### PR DESCRIPTION
**Pull request by acka47**
Resolves #456

### CODE REVIEW

File: schemas/affiliation.json [hunk 1]
```diff
@@ -1,11 +1,14 @@
 {
   "$id": "https://hbz.github.io/lobid/schemas/affiliation.json",
   "$schema": "http://json-schema.org/draft-07/schema#",
-  "title": "Affiliation",
-  "description": "School or organisation, the person is affiliated with.",
+  "title": "Affiliierung",
```
Comment (TobiasNx) [inline schemas/affiliation.json:4]: Better is "(institutionelle) Zugehörigkeit"?

(Im Deutschen gibts zwar Affiliation, aber Affiliierung hab ich noch nie gehört und im Duden finde ich es nicht)

File: schemas/sameAs.json [hunk 2]
```diff
@@ -1,11 +1,14 @@
 {
   "$id": "https://hbz.github.io/lobid/schemas/sameAs.json",
   "$schema": "http://json-schema.org/draft-07/schema#",
-  "title": "Other refrence pages",
+  "title": "Profile anderso",
```
Comment (TobiasNx) [inline schemas/sameAs.json:4]: anderso?

File: schemas/description.json [hunk 3]
```diff
@@ -1,10 +1,13 @@
 {
   "$id": "https://hbz.github.io/lobid/schemas/description.json",
   "$schema": "http://json-schema.org/draft-07/schema#",
-  "title": "Description",
+  "title": "Beschreibung",
   "description": "The description of the object e.g. product, project. MUST have german and english Version",
   "allOf": [
     { "$ref": "https://hbz.github.io/lobid/schemas/localizedString.json" },
     { "required": ["en", "de"]}
```
Comment (TobiasNx) [inline schemas/description.json:8]: If the language is limited to en and ger the required-condition kann be integrated to the localizedString

**Review by TobiasNx — CHANGES_REQUESTED** [review]
Smaller changes needed. Also I would suggest, that if the title is in german also the description should be.

File: schemas/description.json (hunk 3 above)
Comment (acka47) [reply schemas/description.json:8]: I will rather remove the `required` for now so that we can focus on a German version first.

**Comment by acka47** [comment]
I fixed the names, translated the descriptions (we need i18n for JSON schemas to have those schemas and that web form in en/de) and temoved the required languages.

**Pull request merged**